conference.js

```diff
@@ -1664,27 +1664,47 @@ export default {
             return;
         }
 
+        // Create a new presenter track and apply the presenter effect.
         if (!this.localPresenterVideo && !mute) {
-            // create a new presenter track and apply the presenter effect.
             let { height } = this.localVideo.track.getSettings();
-
-            // Workaround for Firefox since it doesn't return the correct width/height of the desktop stream
-            // that is being currently shared.
-            if (!height) {
-                const desktopResizeConstraints = {
+            const { aspectRatio, width } = this.localVideo.track.getSettings();
```

Comment:
you can import height here too, since it's not reassigned and can be a const

Reply:
height is getting reassigned at line # 1707.